src/js/options.js

```diff
@@ -96,7 +96,7 @@ function loadOptions() {
     activate: function (event, ui) {
       // update options page URL fragment identifier
       // to preserve selected tab on page reload
-      window.location.hash = ui.newPanel.attr('id');
+      history.pushState(null, null, "#" + ui.newPanel.attr('id'));
```

Comment:
Sorry, could we make one more change? I think we should use `history.replaceState` instead, since we don't support back button navigation anyway (navigating back in history updates the URL but doesn't change the tab) and it doesn't seem important to support history navigation, easier to just avoid creating new history states.

Reply:
Sure thing, @ghostwords . Just updated.